tests/unit/EntityInfo.spec.ts

```diff
@@ -375,15 +375,19 @@ describe('EntityInfo - Click Tests - Alterations', () => {
 })
 
 describe('EntityInfo - Click Tests - Dissolutions', () => {
+  // mock the mixin to always return True
+  const AllowableActionsMixin: any = {
+    methods: {
+      isAllowed: () => true
+    }
+  }
```

Comment:
Or I could have used `spyOn` to mock just the method and mocked a return value (or implementation).

Well, now we have a choice for future tests :)